.github/workflows/tests.yml

```diff
@@ -16,28 +16,28 @@ jobs:
       matrix:
         os: [ubuntu, windows, macos]
         os_version: [latest]
-        PYTHON_VERSION: ['3.9', '3.10', ]
```

Comment:
Why are we skipping 3.11?

Reply:
In the interest of keeping test matrix not too large, 3.11 is only tested on ubuntu.